### PR TITLE
Update dotnet SDK with account service

### PR DESCRIPTION
**Pull request by tmarkovski**

### CODE REVIEW

**Comment by tmarkovski** [comment]
CI won't pass until we deploy server to staging

File: .github/workflows/build-dotnet.yml [hunk 1]
```diff
@@ -20,10 +20,7 @@ on:
 jobs:
   test:
     name: Test
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: macos-latest
```
Comment (fundthmcalculus) [inline .github/workflows/build-dotnet.yml:23]: We should test on all platforms.

**Review by fundthmcalculus — APPROVED** [review]
See comment

File: .github/workflows/build-dotnet.yml (hunk 1 above)
Comment (tmarkovski) [reply .github/workflows/build-dotnet.yml:23]: We should, but the other platforms don't support `net6.0-ios` target, so we need to separate that out first.

**Pull request merged**